model_analyzer/config/generate/automatic_model_config_generator.py

```diff
@@ -60,6 +63,11 @@ def __init__(self, config, model, client, variant_name_manager,
 
         self._reset_max_batch_size()
 
+        if not self._early_exit_enable:
+            raise TritonModelAnalyzerException(
+                "Early exit disable not supported in automatic model config generator"
```

Comment:
should there be an "is" between disable and supported?

Reply:
done